facebook/Module.md

```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
 ```ballerina
 var response = facebookEP.retrievePost(postId);
 if (response is facebook:Post) {
-    p = response;
-    io:println(p);
+    io:println("Post Details: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success and `error` on failure.
+The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success or an `error` if unsuccessful.
 ```ballerina
 var response = facebookEP.deletePost(postId);
 if (response is boolean) {
-    b = response;
-    io:println(b);
+    io:println("Status: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success and `error` on failure.
+The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success or an `error` if unsuccessful.
 ```ballerina
 var response = facebookEP.getFriendListDetails(userId);
 if (response is facebook:FriendList) {
-    friendList = response; 
-    io:println(friendList); 
+    io:println("Friend List: ", response);
 } else {
-     io:println(response);
+     io:println("Error: ", response);
 }
 ```
 
-The `getPageAccessTokens` function used to get the page access tokens. The `userId` represents the ID of the user. It returns the `AccessTokens` object on success and `error` on failure.
+The `getPageAccessTokens` function used to get the page access tokens. The `userId` represents the ID of the user. It returns the `AccessTokens` object on success or an `error` if unsuccessful.
 ```ballerina
 var response = facebookEP.getPageAccessTokens(userId);
 if (response is facebook:AccessTokens) {
-    list = response; 
-    io:println(accessTokenList); 
+    io:println("Page Access Tokens: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
 ##### Example
 
 ```
 import ballerina/io;
+import ballerina/http;
 import wso2/facebook;
 import ballerina/config;
 
-public function main() {
-    string userAccessToken = config:getAsString("ACCESS_TOKEN");
-    string pageAccessToken = callMethodsWithUserToken(userAccessToken);
-    callMethodsWithPageToken(pageAccessToken);
-}
-
 facebook:FacebookConfiguration facebookConfig = {
     clientConfig:{
         auth:{
             scheme: http:OAUTH2,
-            accessToken:accessToken
+            accessToken: getUserAccessToken()
```

Comment:
Do we need a function? We can directly say `config:getAsString("ACCESS_TOKEN");` right?